.github/workflows/check-flake-files.yml

```diff
@@ -0,0 +1,55 @@
+name: "Check Flake Groups"
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'flakes/**.toml'
+
+jobs:
+  automatic-custom-flakes-check:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+
+    env:
+      RUST_LOG: debug
+    
+    steps:
+
+    - name: Checking out the repository
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
Why? This fetches the entire history.

Reply:
I don't know, was there before just moved it.

Reply:
Wait, wrong PR the moving happens in the next one, sorry for the spoiler
Anyway, I don't know why, will fix it in #433